agent/consul/state/acl_schema.go

```diff
@@ -285,6 +286,19 @@ func bindingRulesTableSchema() *memdb.TableSchema {
 	}
 }
 
+func indexIDFromACLBindingRule(raw interface{}) ([]byte, error) {
+	p, ok := raw.(*structs.ACLBindingRule)
+	if !ok {
+		return nil, fmt.Errorf("unexpected type %T for structs.ACLBindingRule index", raw)
+	}
+	vv, err := uuidStringToBytes(p.ID)
+	if err != nil {
+		return nil, err
+	}
+
+	return vv, err
```

Comment:
```suggestion
	return uuidStringToBytes(p.ID)
```

Reply:
I guess this need to go in enterprise otherwise merge conflict.
Never mind we will clean it next time we change that